src/main/java/org/jabref/logic/importer/fetcher/ArXiv.java

```diff
@@ -116,6 +118,9 @@ public TrustLevel getTrustLevel() {
     }
 
     private List<ArXivEntry> searchForEntries(BibEntry entry) throws FetcherException {
+        entry = (BibEntry) entry.clone();
```

Comment:
Why do you clone the entry?

Reply:
If we do clean up on the original entry, its fields will be changed but I think we needn't change fields here because it is just a search.